fedra.sh

```diff
@@ -0,0 +1,60 @@
+package: FEDRA
+version: rev1523
+tag: rev1523
+
+source: https://github.com/antonioiuliano2/fedra
+
+requires:
+  - GCC-Toolchain
+  - ROOT
+
+--- 
+#!/bin/bash 
+rsync -a $SOURCEDIR/* $BUILDDIR
+
+export FEDRA_INSTALL_DIR=$INSTALLROOT
+
+chmod u+x install.sh
+./install.sh
+
+rm $INSTALLROOT/src/libShower/weights/Energy/volumeSpec_CP/particleSpec_electron/efficiencySpec_MiddleFix/trainrangeSpec_normal/trainrangeSpec_normal #link to a file in meisel folders?!?!
+cp $INSTALLROOT/src/libAnalysis/EdbDecaySearch.h $INSTALLROOT/include/ #copy file missing from include folder
+rsync -aL $INSTALLROOT/* $BUILDDIR/tempdir #copy links as actual files
+rsync -aL $BUILDDIR/tempdir/* $INSTALLROOT #copy links as actual files
+
+rsync -a $INSTALLROOT/src/*/*.pcm $INSTALLROOT/lib
+rsync -a $INSTALLROOT/src/*/*/*.pcm $INSTALLROOT/lib
+rsync -a $INSTALLROOT/macros/rootlogon_root6x.C $INSTALLROOT/macros/rootlogon.C
+rsync -a $INSTALLROOT/macros_root6/*.C $INSTALLROOT/macros/
+
+#missing folders in include
+rsync -aL $INSTALLROOT/src/libVt++/smatrix/include/ $INSTALLROOT/include/smatrix/include
+rsync -aL $INSTALLROOT/src/libVt++/vt++/include/ $INSTALLROOT/include/vt++/include
+rsync -aL $INSTALLROOT/src/libDataConversion/dataIO/ $INSTALLROOT/include/dataIO/include
+
+# Modulefile
+MODULEDIR="$INSTALLROOT/etc/modulefiles"
+MODULEFILE="$MODULEDIR/$PKGNAME"
+mkdir -p "$MODULEDIR"
+cat > "$MODULEFILE" <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0
```

Comment:
Doesn't Fedra need ROOT?

Reply:
Yes, you are right! I did not notice this, since I always tested FEDRA with sndsw, so the ROOT module is already imported.
But a stand-alone FEDRA build would have not worked, without the ROOT module.

I copied the ROOT module loading string from fairroot.sh, so it should be fine:

module load BASE/1.0                                                                            \\
            ROOT/$ROOT_VERSION-$ROOT_REVISION

Reply:
This actually got me the error:
(6):ERROR:105: Unable to locate a modulefile for 'ROOT/-local1'

Reply:
The cvmfs version

Reply:
That counts as "system", since we let alibuild find it using the `prefer_system_check` mechanism.
This is why $ROOT_VERSION is not set, and $ROOT_REVISION has that strange value.

If you use `${ROOT_VERSION:+ROOT/$ROOT_VERSION-$ROOT_REVISION}`, it should be OK, as it will evaluate to nothing if system ROOT is used.

Reply:
Thank you very much! I confirm that using this argument removes the error.